src/pages/faq.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import SEO from '../components/seo'
+import Layout from '../containers/layout'
+import Panel from '../components/Panel'
+
+const FAQPage = () => {
+  return (
+    <Layout>
+      <SEO
+        title="QueerJS - Organizers"
+        description="A meetup for everyone where Queer Speakers take the stage"
+      />
+      <main>
+        <h1 hidden>Welcome to QueerJS</h1>
+        <Panel heading="FAQ"></Panel>
+        <Panel heading="How do I speak at QueerJS?">
+          We'd love for you to speak at a meetup! Please see {' '}  
+          <a
+            href="/speak"
+            rel="noopener noreferrer"
+            title="Speak"
+          >
+            Speaking at QueerJS
+          </a> for more information about how you can propose a talk to a future iteration of QueerJS.
+        </Panel>
+        <Panel heading="How can I get involved?">
+          There are a lot of ways you could get involved with QueerJS! We're always looking for new locations to
+          host meetups, as well as folks on the ground to act as local organizers and MCs.
+          <br/><br/>
+          Involvement might involve working with your employers to provide a space, organizing catering for meetup
```

Comment:
I'm the most annoying copy editor but can we change `Involvement might involve`? Maybe to something like `Involvement might look like` or `Involvement might mean` or something along those lines?

Reply:
oh derp this does not read well you are v correct lol